config/default.js

```diff
@@ -10,12 +10,18 @@
  */
 var defer = require('config/defer').deferConfig;
 module.exports = {
+  secrets: {
+    auth: {
+      secret: 'changeThis!'
+    }
+  },
   server: {
+    debug: '',
     port: 3030,
     version: '0.1.1',
     /* Configuration can be built at runtime.  Deferred functions cannot be replaced.  Deferred functions cannot alter anything (must be immutable). */
     build : defer(function ()  {
-      return this.server.version + ' ' + Math.random()
+      return this.server.version + ' ' + Math.random();
```

Comment:
Why is Math.random() used here? Wouldn't that make the end result something like this:

0.1.1 4

0.1.1 being the version, and 4 being a random number?